frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -897,31 +903,35 @@ export const insightLogic = kea<insightLogicType>([
             }
         },
         abortQuery: async ({ queryId }) => {
-            const { currentTeamId } = values
+            try {
+                const { currentTeamId } = values
 
-            await api.create(`api/projects/${currentTeamId}/insights/cancel`, { client_query_id: queryId })
+                await api.create(`api/projects/${currentTeamId}/insights/cancel`, { client_query_id: queryId })
 
-            const duration = performance.now() - values.queryStartTimes[queryId]
-            await captureTimeToSeeData(values.currentTeamId, {
-                type: 'insight_load',
-                context: 'insight',
-                primary_interaction_id: queryId,
-                query_id: queryId,
-                status: 'cancelled',
-                time_to_see_data_ms: Math.floor(duration),
-                insights_fetched: 0,
-                insights_fetched_cached: 0,
-                api_response_bytes: 0,
-                insight: values.activeView,
-            })
+                const duration = performance.now() - values.queryStartTimes[queryId]
+                await captureTimeToSeeData(values.currentTeamId, {
+                    type: 'insight_load',
+                    context: 'insight',
+                    primary_interaction_id: queryId,
+                    query_id: queryId,
+                    status: 'cancelled',
+                    time_to_see_data_ms: Math.floor(duration),
+                    insights_fetched: 0,
+                    insights_fetched_cached: 0,
+                    api_response_bytes: 0,
+                    insight: values.activeView,
+                })
+            } catch (e) {
+                console.warn('Failed cancelling query', e)
```

Comment:
Every so often this still causes the tests to fail. I think this is only a problem in tests since we don't see it in sentry